src/Properties/ModelRelationsExtension.php

```diff
@@ -87,8 +88,45 @@ public function getProperty(ClassReflection $classReflection, string $propertyNa
             return new ModelProperty($classReflection, IntegerRangeType::createAllGreaterThanOrEqualTo(0), new NeverType(), false);
         }
 
-        $method = $classReflection->getMethod($propertyName, new OutOfClassScope());
+        $methodReflection = $classReflection->getMethod($propertyName, new OutOfClassScope());
+
+        // @phpstan-ignore-next-line
```

Comment:
Can you use `@phpstan-ignore` instead and perhaps provide a reason? I assume something like this:

```php
// @phpstan-ignore always.true (we support different Laravel versions)
```

Reply:
I tried that and it didn't work because the identifier changes depending on the Laravel version (e.g., `always.false).

However, I've reworked all of the `phpstan-ignore-[next-]line` tags to use the new syntax with comments on why they're being ignored.